robot/client/client_options.go

```diff
@@ -56,6 +58,14 @@ func newFuncRobotClientOption(f func(*robotClientOpts)) *funcRobotClientOption {
 	}
 }
 
+// WithModName attaches a unary interceptor that attaches the module name for each outgoing gRPC
+// request.
```

Comment:
```suggestion
// request. Should only be used in Viam module library code.
```
Don't really care about the wording of the messaging, but maybe just a line to indicate that Golang SDK users (not that there are many) shouldn't really touch this option.

Reply:
Done